index.js

```diff
@@ -904,12 +938,24 @@ instance.prototype.feedback = function(feedback) {
 			return { color: feedback.options.fg, bgcolor: feedback.options.bg };
 		}
 	}
+
 	if (feedback.type === 'scene_item_active')  {
 		if ((self.states[feedback.options.source] === true)) {
 			return { color: feedback.options.fg, bgcolor: feedback.options.bg };
 		}
 	}
 
+	if (feedback.type === 'scene_item_active_in_scene') {
+		let scene = self.scenes[feedback.options.scene];
+		if (scene) {
```

Comment:
If you're doing a check here, I'd prefer you check for the sources array too: `if (scene && scene.sources) {`

Reply:
I've updated it.